src/domain/buildingBlocks/Pluto.ts

```diff
@@ -121,76 +96,6 @@ export interface Pluto {
    */
   getDIDPrivateKeysByDID(did: DID): Promise<Array<PrivateKey>>;
 
-  /**
-   * Retrieve private key for a given key ID.
-   */
-  getDIDPrivateKeyByID(id: string): Promise<PrivateKey | null>;
-
-  /**
-   * Retrieve all stored DID pairs (DIDComm connections).
-   */
-  getAllDidPairs(): Promise<Array<DIDPair>>;
-
-  /**
-   * Retrieve a DID pair containing a given DID as either host or receiver.
-   */
-  getPairByDID(did: DID): Promise<DIDPair | null>;
-
-  /**
-   * Retrieve a DID pair by a given pair name.
-   */
-  getPairByName(name: string): Promise<DIDPair | null>;
```

Comment:
we never seem to retrieve a DIDPair from Pluto.
ConnectionsManager caches them after saving.

Reply:
I'd report this as a bug I guess. Why? 

Users are adding the didPair for connection to the database when connection is established. User's should be able to revoke an existing connection on their side and right now the only way would be to do it by storing the didPairs. 
Then, if the user is getting a message from a "valid" connection which he has removed, then the messages should be discarded and not stored.